pom.xml

```diff
@@ -197,7 +197,7 @@ THE SOFTWARE.
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>git</artifactId>
-            <version>${git-plugin.version}</version>
+            <version>3.9.1</version>
```

Comment:
You should be updating `git-plugin.version` in `properties` rather than just changing it here, fwiw.

Reply:
Roger that. Done.